setup.py

```diff
@@ -113,7 +117,7 @@ def build_extension(self, ext):
         license='BSD-3C',
         packages=find_packages(),
         include_package_data=True,
-        ext_modules=[CMakeExtension('gau2grid.libgg')],
+        ext_modules=[CMakeExtension('gau2grid.gg')],
```

Comment:
Aha, makes sense.

Reply:
Actually, what does this do? I just changed it to be consistent -- didn't seem to have influence.

Reply:
It used to change the object name, I guess this was removed at some point.